src/components/NewBeer.js

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+
+const NewBeer = () => {
+
+  const [name, setName] = useState('');
+  const [tagline, setTagline] = useState('');
+  const [description, setDescription] = useState('');
+  const [firstBrewed, setFirstBrewed] = useState('');
+  const [brewersTips, setBrewersTips] = useState('');
+  const [attenuationLevel, setAttenuationLevel] = useState(0);
+  const [contributedBy, setContributedBy] = useState('');
+  const navigate = useNavigate();
+
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    const newBeer = {
+       name,
+       tagline,
+      description,
+      attenuationLevel,
```

Comment:
Here it has to be `attenuation_level: attenuationLevel`, same as you did with the first_brewed etc. because it's called attenuation_level in the Beer.Model in the backend. attenuation_level is not saved when creating a beer with just `attenuationLevel`

Reply:
oh yeah you right,,i didnt notice,,< thank you,<